ledger/participant-integration-api/src/main/scala/platform/store/backend/common/UserManagementStorageBackendTemplate.scala

```diff
@@ -83,7 +84,7 @@ object UserManagementStorageBackendTemplate extends UserManagementStorageBackend
     SQL"""SELECT user_id, primary_party
           FROM participant_users
           $whereClause
-          ORDER BY user_id
+          ORDER BY user_id #${queryStrategy.binaryCollation}
```

Comment:
If query collation does not match index collation it will boil down to a table scan. The proper approach is probably set the collation for the column, and not setting it for the query (at least for postgresql, we need to figure this out for Oracle as well)